code/modules/clothing/gloves/color.dm

```diff
@@ -85,6 +85,18 @@
 	. = ..()
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
+	if(Holiday == APRIL_FOOLS_DAY)
+		new /obj/item/clothing/gloves/fyellow/insulted(src)
+		qdel(src)
```

Comment:
This is actually gonna create an infinite loop since insulted gloves are a subtype of fgloves. You'll have to either remove the ..() from insulted gloves New() and suffer the unforseen consequences or find a different approach